docs/getting-started/sensor-full-disk-access.asciidoc

```diff
@@ -13,7 +13,7 @@ The behavior of the Endgame sensor and Elastic Endpoint differs based on your ma
 
 **Endgame Sensor**
 
-- `10.13, 10.14, 10.15`: Users cannot proceed with installation without first granting the sensor the ability to load a <<enable-kernel-extension,kernel extension>>. During installation, you will be prompted to go to **System preferences** and approve loading the kernel. Upon approval, installation proceeds.
+//- `10.15`: Users cannot proceed with installation without first granting the sensor the ability to load a <<enable-kernel-extension,kernel extension>>. During installation, you will be prompted to go to **System preferences** and approve loading the kernel. Upon approval, installation proceeds.
```

Comment:
@ymao1 can we remove this information in the [7.17 docs](https://www.elastic.co/guide/en/security/7.17/sensor-full-disk-access.html)?

Reply:
This part of document describes endgame sensor, correct? If so, there's no need to remove, this paragraph looks fine.

Reply:
I meant this paragraph in green looks fine and no need to remove the paragraph because it is correct endgame sensor behavior:
> `10.15`: Users cannot proceed with installation without first granting the sensor the ability to load a <<enable-kernel-extension,kernel extension>>. During installation, you will be prompted to go to **System preferences** and approve loading the kernel. Upon approval, installation proceeds.

Reply:
Ok thanks for confirming. The 7.17 Endgame sensor docs are organized a bit differently from the 8.x docs so I wanted to double-check with you before altering anything.

Reply:
I'll remove the comment code and keep the paragraph in the docs.

Reply:
```suggestion
- `10.15`: Users cannot proceed with installation without first granting the sensor the ability to load a <<enable-kernel-extension,kernel extension>>. During installation, you will be prompted to go to **System preferences** and approve loading the kernel. Upon approval, installation proceeds.
```

Reply:
@jmikell821 the link in this paragraph points to docs for enabling the [Elastic Endpoint kernel](https://www.elastic.co/guide/en/security/7.17/install-endpoint.html#enable-kernel-extension). Do you happen to know if this reference is correct or if it should be pointing towards other docs that are more specific to the granting FDA to the Endgame sensor?

Reply:
Need to re-add the kernel extension docs to this PR so the checks resolve. After do that, can merge this PR.

Reply:
Fixed in [b2bae01](https://github.com/elastic/security-docs/pull/3698/commits/b2bae0145d5c1e62dd094b2018773c2b484f2fd9).